app/src/androidTest/java/mollie/tictactoe/MainActivityTest.java

```diff
@@ -26,29 +26,33 @@
 
     @Test
     public void canWinAGame() {
-        onView(withId(R.id.left_top_corner)).perform(click());
-        onView(withId(R.id.left_centre)).perform(click());
-        onView(withId(R.id.top_centre)).perform(click());
-        onView(withId(R.id.centre_button)).perform(click());
-        onView(withId(R.id.right_top_corner)).perform(click());
+        findButtonById(R.id.left_top_corner).perform(click());
+        findButtonById(R.id.left_centre).perform(click());
+        findButtonById(R.id.top_centre).perform(click());
+        findButtonById(R.id.centre_button).perform(click());
+        findButtonById(R.id.right_top_corner).perform(click());
         onView(withText("X wins!"))
                 .inRoot(withDecorView(not(is(mActivityRule.getActivity().getWindow().getDecorView()))))
                 .check(matches(isDisplayed()));
     }
 
     @Test
     public void canDrawAGame() {
-        onView(withId(R.id.left_top_corner)).perform(click());
-        onView(withId(R.id.centre_button)).perform(click());
-        onView(withId(R.id.left_centre)).perform(click());
-        onView(withId(R.id.left_bottom)).perform(click());
-        onView(withId(R.id.right_top_corner)).perform(click());
-        onView(withId(R.id.top_centre)).perform(click());
-        onView(withId(R.id.centre_bottom)).perform(click());
-        onView(withId(R.id.right_bottom)).perform(click());
-        onView(withId(R.id.right_centre)).perform(click());
+        findButtonById(R.id.left_top_corner).perform(click());
```

Comment:
findButtonById is pretty neat. 
How about taking it one step further with clickOnButton(id)?